components/esm/cellref.cpp

```diff
@@ -4,13 +4,34 @@
 #include "esmreader.hpp"
 #include "esmwriter.hpp"
 
-void ESM::CellRef::load (ESMReader& esm, bool wideRefNum)
+void ESM::RefNum::load (ESMReader& esm, bool wide)
 {
-    loadId(esm, wideRefNum);
+    if (wide)
+        esm.getHNT (*this, "FRMR", 8);
+    else
+        esm.getHNT (mIndex, "FRMR");
+}
+
+void ESM::RefNum::save (ESMWriter &esm, bool wide, const std::string& tag) const
```

Comment:
During the save procedure is RefNum is saved with a MVRF tag in some cases (see savingstages.cpp). I am actually not quite sure why the same isn't true for the load part.